pkgs/applications/graphics/inkscape/extensions/textext.nix

```diff
@@ -0,0 +1,72 @@
+{ lib
+, fetchFromGitHub
+, runCommand
+, inkscape
+, texlive-combined
+, python3
+}:
+
+python3.pkgs.buildPythonPackage rec {
+  pname = "textext";
+  version = "1.3.1";
+
+  src = fetchFromGitHub {
+    owner = "textext";
+    repo = "textext";
+    rev = "1.3.1";
+    sha256 = "0sq2gwlklgz55lqxrjnivjl8lmakr2c7dm3q10xlr0slhilrs7sv";
+  };
+
+  patches = [
+    # Make sure we can point directly to pdflatex in the extension,
+    # instead of relying on the PATH (which might not have it)
+    (runCommand "textext-paths.patch" {} ''
```

Comment:
Sorry, meant `substituteAll` function in Nixpkgs:

https://github.com/NixOS/nixpkgs/blob/291b3ff5af268eb7a656bb11c73f2fe535ea2170/pkgs/desktops/gnome/core/gnome-shell/default.nix#L80-L84

(Will need placeholders marked with `@` in the patch file.

Reply:
Ahh, TIL :)